webhook_notification_test.go

```diff
@@ -2,9 +2,35 @@ package braintree
 
 import (
 	"encoding/base64"
+	"io/ioutil"
+	"net/http"
+	"strings"
 	"testing"
 )
 
+func TestWebhookParseRequest(t *testing.T) {
+	t.Parallel()
+
+	gateway := New(Sandbox, "mid", "sz9g7zhxz8838v7h", "0c809a2d2e8f4e4c817900ff441c9554")
```

Comment:
We use the `testGateway` for integration tests to hit an account we have configured with plans, and the needed merchant accounts. This is a unit test and the public/private key were used to create the test setup values. If we change the account we use for integration testing or roll it's keys it'd be best if we didn't have to update this test since it's unrelated to the integration account anyway.

The values I've used for the public and private keys I generated in sandbox by adding a new key to an account, sent a webhook to `requestb.in`, then destroyed the keys, so they don't point to an actual account.